realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/CredentialsTests.kt

```diff
@@ -30,13 +35,15 @@ import kotlin.test.assertFailsWith
 class CredentialsTests {
 
     private lateinit var app: App
-
+    private lateinit var admin: ServerAdmin
 
     companion object {
+
         @BeforeClass
         @JvmStatic
         fun setUp() {
             Realm.init(InstrumentationRegistry.getInstrumentation().targetContext)
+            RealmLog.setLevel(LogLevel.DEBUG)
```

Comment:
This should probably be removed unless you reset it to WARN in `tearDown`. Otherwise it will impact all other tests running after this.

Reply:
I always forget to remove this sucker... :-(

Reply:
Maybe we could clear it in app.close() but I do not know if there would be a need to set it across all tests. If so you have to set it back to the level that was before.